Podcast/SubscriptionsCollectionViewFlowLayout.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  SubscriptionsCollectionViewFlowLayout.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 3/6/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class SubscriptionsCollectionViewFlowLayout: UICollectionViewFlowLayout {
```

Comment:
I personally think this adds clarity and it is what we have done in every other collection view cell but I can change it if necessary